doc/changelog.rst

```diff
@@ -16,6 +16,7 @@ PyMongo 4.5 brings a number of improvements including:
   and :meth:`~pymongo.command_cursor.CommandCursor.try_next` to support
   executing an arbitrary command that returns a cursor.
 - cryptography 2.5 or later is now required for :ref:`OCSP` support.
+- Improved bson encoding performance (`PYTHON-3729`_, `PYTHON-3797`_, `PYTHON-3816`_, `PYTHON-3817`_, `PYTHON-3820`_, `PYTHON-3824`_, and `PYTHON-3846`_).
```

Comment:
“encoding and decoding performance”? Can we add any metrics here?

Reply:
good call, and yeah!